.github/workflows/ci.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
 
       - name: Run protolint
         # Only use hashes here, as we are passing the github token, we want to
-        # make sure updates are done consciously to avoid security issues if the
+        # make sure updates are done conciously to avoid security issues if the
```

Comment:
`consciously` is the correct spelling.

Reply:
OK, this is really weird, because the protolint code in repo-config was copied from here, I'm not sure how that typo sneaked in. Will fix here and in repo-config.